staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/width.go

```diff
@@ -105,6 +108,15 @@ func (e *workEstimator) estimate(r *http.Request, flowSchemaName, priorityLevelN
 	switch requestInfo.Verb {
 	case "list":
 		return e.listWorkEstimator.EstimateWork(r, flowSchemaName, priorityLevelName)
+	case "watch":
+		// WATCH supports `SendInitialEvents` option, which effectively means
+		// that is starts with sending of the contents of a corresponding LIST call.
+		// From that perspective, given that the watch only consumes APF seats
+		// during its initialization (sending init events), its cost should then
+		// be computed the same way as for a regular list.
+		if !utilfeature.DefaultFeatureGate.Enabled(features.WatchList) {
```

Comment:
I am surprised at the negation here.  Don't we want to use the list estimator exactly when the feature gate is _enabled_?